pyanaconda/modules/payloads/payload/flatpak/flatpak_manager.py

```diff
@@ -261,6 +281,29 @@ def _create_flatpak_transaction(self, installation):
 
         return transaction
 
+    # FlatpakTransaction.set_no_pull() does not leave sideload
```

Comment:
This sounds like a bug - is there an upstream bugreport for this we could link from there ?

Reply:
It is and it is filed already. Mentioned above and adding link to the comment.